OpenRA.Mods.Common/EditorBrushes/EditorDefaultBrush.cs

```diff
@@ -84,6 +84,11 @@ public void ClearSelection()
 			}
 		}
 
+		public void UpdateSelection()
+		{
+			SelectionChanged?.Invoke();
```

Comment:
The undo/redo needs to trigger the SelectionChanged event, so I've added this public function to invoke it. I'm not sure if it's the best way of going about it and feedback would be appreciated.

Reply:
you could change the Selection variable to always invoke selection changed event when set

```csharp
EditorSelection selection = new();
public EditorSelection Selection
{
	get => selection;
	set
	{
		selection = value;
		SelectionChanged?.Invoke();
	}
}
```